docs/docs/spark-ddl.md

```diff
@@ -566,3 +566,85 @@ Tags can be removed via the `DROP TAG` sql
 ```sql
 ALTER TABLE prod.db.sample DROP TAG `historical-tag`
 ```
+
+### Iceberg Views in Spark
+
+#### Creating a View
+Creating a view in its simplest form can be done as shown below:
+```sql
+CREATE VIEW <viewName> AS SELECT * from <tableName>
+```
+
+Using `IF NOT EXISTS` prevents the SQL from failing in case the view already exists:
+```sql
+CREATE VIEW IF NOT EXISTS <viewName> AS SELECT * from <tableName>
+```
+
+It is also possible to use a modified schema for the view, where each column has an alias that is different from the source table and carries a comment.
+Additionally, the view itself can have a comment as can be seen below:
+```sql
+CREATE VIEW <viewName> (ID COMMENT 'Unique ID', ZIP COMMENT 'Zipcode')
+    COMMENT 'View Comment'
+    AS SELECT id, zip FROM <tableName>
+```
+
+#### Creating a View with Properties
+
+A view can be created with properties by using `TBLPROPERTIES`:
+```sql
+CREATE VIEW <viewName>
+    TBLPROPERTIES ('key1' = 'val1', 'key2' = 'val2')
+    AS SELECT * from <tableName>
+```
+
+These properties can be displayed using:
+```sql
+SHOW TBLPROPERTIES <viewName>
+```
+
+#### Replacing a View
+
+Updating a view's schema, its properties, or the underlying SQL can be achieved as shown below:
+```sql
+CREATE OR REPLACE <viewName> (updated_id COMMENT 'updated ID')
+    TBLPROPERTIES ('key1' = 'new_val1')
+    AS SELECT id FROM <tableName>
+```
+
+#### Setting/removing View properties
+
+Properties can be set via the following SQL:
+```sql
+ALTER VIEW <viewName> SET TBLPROPERTIES ('key1' = 'val1', 'key2' = 'val2')
+```
+
+Properties can be removed using:
+```sql
+ALTER VIEW <viewName> UNSET TBLPROPERTIES ('key1')
+```
+
+#### Showing available Views
+
+The below SQL will list all views in the currently set namespace:
+```sql
+SHOW VIEWS
+```
+
+The below SQL will list all available views in the defined catalog/namespace
+```sql
+SHOW VIEWS IN <catalog|namespace>
+```
+
+#### Showing the CREATE stmt of a View
+
+```sql
+SHOW CREATE TABLE <viewName>
```

Comment:
`Table` keyword here and `TBLPROPERTIES` keywords above looks weird for view. 

But thats how the Spark syntax is also maintained I guess. 
https://spark.apache.org/docs/3.0.0-preview/sql-ref-syntax-ddl-create-view.html#create-view

Reply:
yes that's the official Spark syntax